src/sentry/integrations/bitbucket/repository.py

```diff
@@ -27,7 +27,7 @@ def get_installation(self, integration_id, organization_id):
 
         return integration_model.get_installation(organization_id)
 
-    def validate_config(self, organization, config):
+    def add_repository_data(self, organization, config):
```

Comment:
@markstory it can. I guess I just thought because we're passing in a config into the method already that I was adding things to it rather than creating it anew altogether.

Reply:
ok. I don't have strong convictions around this. What you have is an improvement :smile: